### ISSUE TITLE
New module automapper dependency injection error working

### ISSUE THREAD

**Issue by mkmita** [abpframework/abp#2578]
Module automapper not working and giving autofac dependency injection error.

To make it work I made changed in file TestModuleWebModule.cs
changed code below
```
Configure<AbpAutoMapperOptions>(options =>
            {
                options.AddMaps<PlacesWebModule>(validate: true);
            });
```
to this
```
context.Services.AddAutoMapperObjectMapper<TestModuleWebModule>();
            Configure<AbpAutoMapperOptions>(options =>
            {
                options.AddProfile<TestModuleWebAutoMapperProfile>(validate: true);
            });
```

not sure this is bug or not but its very annoying when you having a framework and templates are not working out of the box

**Comment by hikalkan** [comment]
Could not repeat with the current template source code (on the dev branch). What is your ABP version? Also, please share your stacktrace in such cases.

**Comment by maliming** [comment]
Has it been fixed by https://github.com/abpframework/abp/commit/474fc95824405143e02e84d696c5654b0c26bbaf?

**Comment by hikalkan** [comment]
No. Please try with a template downloaded with the latest version from abp.io web site.

**Comment by maliming** [comment]
hi @mkmita 
Couldn't reproduce your problem, Please share more details.
Are you trying to inject `IObjectMapper<>`?
Do you read the documentation of the **Object-To-Object-Mapping**?
https://docs.abp.io/en/abp/latest/Object-To-Object-Mapping

Error stack info are always more helpful than text descriptions.

**Comment by mkmita** [comment]
@maliming software was not able to resolve mapper in mvc module PageModel.

**Comment by maliming** [comment]
> software was not able to resolve mapper in mvc module PageModel. 

Can you share the stack of exception? ( ´･･)

I think you can easily use abp cli to create a project that reproduces the problem, please share it if possible.

If there is a problem with the template, we will fix it in the 2.0 milestone. Thanks.

**Comment by maliming** [comment]
hi @mkmita  Any news?
